crowd_anki/importer/import_dialog.py

```diff
@@ -0,0 +1,189 @@
+from collections import namedtuple
+from dataclasses import dataclass, field
+from enum import Enum
+
+from aqt.qt import *
+from typing import List, Tuple, Dict
+
+from .import_ui import Ui_Dialog as ConfigUI
+from ..config.config_settings import ConfigSettings
+from ..utils.constants import UUID_FIELD_NAME
+
+from ..utils.utils import string_cs_to_list
+
+
+@dataclass
+class ConfigEntry:
+    config_name: str
+    default_value: any
+
+
+@dataclass
+class PersonalFieldsHolder:
+    personal_fields: Dict[str, List[str]] = field(init=False, default_factory=lambda: dict())
+
+    def has_pf(self, field_name, *keys):
```

Comment:
don't do abbreviations. this should probably be `contains_field`

Reply:
also should `field_name`  be `model_name` actually?

Reply:
oh wait, this checks if any of the models given contains the specified personal field. and we need it because we support both uids and the names now?  
I think supporting just the names is fine they should be unique withing given export

Reply:
on looking how it's used, I think the interface should be something like `is_personal_field(model, field_name`